Ghidra/RuntimeScripts/macOS/support/macos_debugger_lldb_build_from_brew.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+# This script will download lldb from homebrew and
+# build the Ghidra JNI bindings for this version of
+# lldb. It will then patch your Ghidra distribution
+# to use these built libraries.
+
+set -ex
+
+if [ -z "${GHIDRA_INSTALL_DIR}" ]; then
+	echo "Please set the GHIDRA_INSTALL_DIR environment variable to your Ghidra install location"
+	exit 1
+fi
+
+if [ ! -z "${GHIDRA_INSTALL_DIR}" ]; then
+	pushd "${GHIDRA_INSTALL_DIR}/Ghidra/Debug/Debugger-swig-lldb"
+fi
+
+# Pin to 14, as this is what Ghidra's built in bindings are built against
+LLVM_VERSION="14"
+
+# Install llvm and unpack the source code for this version, patched
+# with the brew patches
+brew install llvm@${LLVM_VERSION}
+
+LLVM_TEMP_DIR=$(mktemp -d)
+
+# Download the source code brew used to build llvm, including
+# brew specific patches.
+brew unpack --patch --destdir ${LLVM_TEMP_DIR} llvm@${LLVM_VERSION}
+export LLVM_HOME="$(echo ${LLVM_TEMP_DIR}/llvm@${LLVM_VERSION}-*)"
+
+# Set the appropriate build variables to link and compile the
+# liblldb-java library below.
+BREW_LLVM="$(brew --prefix llvm@${LLVM_VERSION})"
+export LDFLAGS="-L${BREW_LLVM}/lib/c++ -Wl,-rpath,${BREW_LLVM}/lib/c++,-L${BREW_LLVM}/lib"
+export PATH="${BREW_LLVM}/bin:$PATH"
+export CPPFLAGS="-I${BREW_LLVM}/include"
+
+export LLVM_BUILD="$(echo ${BREW_LLVM})"
+
+# Build native components
+gradle buildNatives
+
+# Build only the library required for our architecture.
```

Comment:
Another clarification, though if there is a better way to do this with gradle I would like to switch.